hamlpy/test/test_attributes.py

```diff
@@ -106,9 +106,16 @@ def test_read_attribute_dict(self):
         assert dict(self._parse("{class: 'test\u1234'}")) == {'class': 'test\u1234'}
 
         # html style dicts
+        assert dict(self._parse("()")) == {}
+        assert dict(self._parse("(   )")) == {}
+
+        assert dict(self._parse(
+            "(disabled :class='test' data-number = 123\n 'foo'=\"bar\")"
+        )) == {'disabled': None, 'class': 'test', 'data-number': '123', 'foo': 'bar'}
+
         assert dict(self._parse(
-            "(:class='test' :data-number = 123\n foo=\"bar\")"
-        )) == {'class': 'test', 'data-number': '123', 'foo': 'bar'}
+            "(:class='test' data-number = 123\n 'foo'=\"bar\"  \t   disabled)"
```

Comment:
That was how you made a hash (dict) in Ruby before 1.9 apparently http://effectif.com/ruby/update-your-project-for-ruby-19-hash-syntax

Agreed on disallowing this - have recorded that at https://github.com/nyaruka/django-hamlpy/issues/43